lib/make/numbers.ts

```diff
@@ -205,7 +220,7 @@ export class numbers {
    */
   static interesting(): number {
     return random.choose([
-      [8, [-128, -1, 0, 1, 16, 32, 64, 100, 127, 255]],
```

Comment:
why remove this? you didn't remove 65535 from the 16 bit cases below, etc.

Reply:
You're right.  I'll remove the remaining numbers outside of the bounds.